bases/next.json

```diff
@@ -6,15 +6,17 @@
     "lib": ["dom", "dom.iterable", "esnext"],
     "allowJs": true,
     "skipLibCheck": true,
-    "strict": false,
```

Comment:
https://nextjs.org/docs/api-reference/create-next-app

`create next-app --typescript ` was used to generate the data.

Reply:
In the latest `tsconfig`, `"strict": true` 😰